CHANGELOG.md

```diff
@@ -1,3 +1,51 @@
+# 1.0.10-rc1 (2020-06-23)
+## Edge Agent
+### Features
+* Update codebase to dotnet 3.1. [f87a18a](https://github.com/Azure/iotedge/commit/f87a18a487ea0c05752254aaba04a4f89028120a)
+* Install Trust Bundle in EdgeAgent and EdgeHub. [4f85dcc](https://github.com/Azure/iotedge/commit/4f85dcc7d3fd6d4772d4a9b86ec3ecad651938fd)
+* Metrics use RocksDB [0d2cf76](https://github.com/Azure/iotedge/commit/0d2cf76ea1da561252a1100b7da5b4f0f5f1d471)
+* Metrics Upload to IoTHub [eff5c85](https://github.com/Azure/iotedge/commit/eff5c859e101d14eda85b80513136a2f3a473892)
```

Comment:
Recommend clarifying what exactly this means

Reply:
These are commit comments. Again, if you want more detail, the commit hash is there for your consumption to look at.

Reply:
If I am a customer I wouldn't really want to do this though. I think most commit messages are fine here but we usually have to touch some up to be presentable.